driver/network_driver.go

```diff
@@ -286,12 +286,12 @@ func (d NetworkDriver) CreateEndpoint(request *network.CreateEndpointRequest) (*
 		// the request if the profile already exists.
 		profile := &api.Profile{
 			Metadata: api.ProfileMetadata{
-				Name: networkData.Name,
-				Tags: []string{networkData.Name},
+				Name:   networkData.Name,
+				Labels: map[string]string{"projectcalico.org/group": networkData.Name},
```

Comment:
I'm not convinced this is the correct label to be using.

This is not logically equivalent to the tag we had before, since now it's only possible to belong to a single Profile (whereas with tags there could be multiple).

I think this means we either need to:
- Encode the profile name into the _key_ of the label rather than the value.
- Just keep using tags.